HelpSource/Classes/NamedControl.schelp

```diff
@@ -5,7 +5,7 @@ related:: Classes/ControlName, Classes/Control
 
 description::
 
-A NamedControl directly combines a ControlName and a Control UGen conveniently. Also this makes it safe even if several identical controls exist (see example below).
+A NamedControl directly combines a ControlName and a Control UGen conveniently. It can be saved to a var to send the control value to different parts of the function or SynthDef.
```

Comment:
Unfortunately, this is a misunderstanding. The sentence "Also this makes it safe even if several identical controls exist (see example below)." meant that you can use the same symbol several times.

Your attempt to correct it indeed indicates that it isn't clear enough, so thank you.
Would you prefer to improve or start a new one?